### PR TITLE
Fix the operand type of the atomic store instruction

### PR DESCRIPTION
**Pull request by AMD-dwang**
In IR verification, Atomic store and load operand must be integer,pointer, or
floating point type. For vector type, we need to spilt the each compent.

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit bac5e50d8fbfdb22236e295dd15de85c8c89f7ca
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev a8cf07c212059bae9d018c05fc896e1115a2f077</li>
<li>SPVGEN: dev 4846370016de9775b7518b51c08260387a7367c4</li>
<li>XGL: dev d3cc9bf1baae76359c2ebab6e7122e6b9c428455</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 20799acf951a8e0b79d95154ecfca851a2f63b98</li>
</ul></details>

<details><summary>CTS tests (Failed: 161/238988)</summary><ul><li>Built with version 1.2.5.1</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 31312/59723 (52.4%)</li>
<li><details><summary>Failed: 40/59723 (0.1%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.spec_constant.compute.composite.array.array_bool
Stack trace: Fail

FAILURE: dEQP-VK.pipeline.spec_constant.graphics.vertex.composite.array.array_bool
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.different_matrix_layout
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.different_matrix_strides
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.nested_arrays_different_inner_stride
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.nested_arrays_different_strides
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.ssbo_to_ubo
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.two_arrays_different_stride_1
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.ubo_to_ssbo
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.2_level_array.scalar.mat3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.3_level_array.scalar.column_major_mat2x3_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.3_level_array.std140.mat3x2
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.basic_unsized_array.scalar.mat3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.scalar.column_major_mat2x3_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.scalar.mat4x3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.std140.mat4x2
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_basic_types.per_block_buffer.scalar
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_nested_struct.per_block_buffer.scalar_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_nested_struct.single_buffer.scalar_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.random.16bit.all_shared_buffer.40
Stack trace: Fail
...
```
</p></details></li>

<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 31312/59723 (52.4%)</li>
<li><details><summary>Failed: 40/59723 (0.1%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.spec_constant.compute.composite.array.array_bool
Stack trace: Fail

FAILURE: dEQP-VK.pipeline.spec_constant.graphics.vertex.composite.array.array_bool
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.different_matrix_layout
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.different_matrix_strides
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.nested_arrays_different_inner_stride
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.nested_arrays_different_strides
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.ssbo_to_ubo
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.two_arrays_different_stride_1
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.ubo_to_ssbo
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.2_level_array.scalar.mat3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.3_level_array.scalar.column_major_mat2x3_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.3_level_array.std140.mat3x2
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.basic_unsized_array.scalar.mat3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.scalar.column_major_mat2x3_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.scalar.mat4x3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.std140.mat4x2
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_basic_types.per_block_buffer.scalar
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_nested_struct.per_block_buffer.scalar_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_nested_struct.single_buffer.scalar_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.random.16bit.all_shared_buffer.40
Stack trace: Fail
...
```
</p></details></li>

<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 31475/59819 (52.6%)</li>
<li><details><summary>Failed: 41/59819 (0.1%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.creation_feedback.graphics_tests.vertex_stage_fragment_stage_delayed_destroy
Stack trace: Fail

FAILURE: dEQP-VK.pipeline.spec_constant.compute.composite.array.array_bool
Stack trace: Fail

FAILURE: dEQP-VK.pipeline.spec_constant.graphics.vertex.composite.array.array_bool
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.different_matrix_layout
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.different_matrix_strides
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.nested_arrays_different_inner_stride
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.nested_arrays_different_strides
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.ssbo_to_ubo
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.two_arrays_different_stride_1
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.ubo_to_ssbo
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.2_level_array.scalar.mat3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.3_level_array.scalar.column_major_mat2x3_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.3_level_array.std140.mat3x2
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.basic_unsized_array.scalar.mat3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.scalar.column_major_mat2x3_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.scalar.mat4x3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.std140.mat4x2
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_basic_types.per_block_buffer.scalar
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_nested_struct.per_block_buffer.scalar_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_nested_struct.single_buffer.scalar_comp_access
Stack trace: Fail
...
```
</p></details></li>

<li>Skipped: 28303/59819 (47.3%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 34953/59723 (58.5%)</li>
<li><details><summary>Failed: 40/59723 (0.1%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.spec_constant.compute.composite.array.array_bool
Stack trace: Fail

FAILURE: dEQP-VK.pipeline.spec_constant.graphics.vertex.composite.array.array_bool
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.different_matrix_layout
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.different_matrix_strides
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.nested_arrays_different_inner_stride
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.nested_arrays_different_strides
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.ssbo_to_ubo
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.two_arrays_different_stride_1
Stack trace: Fail

FAILURE: dEQP-VK.spirv_assembly.instruction.spirv1p4.opcopylogical.ubo_to_ssbo
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.2_level_array.scalar.mat3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.3_level_array.scalar.column_major_mat2x3_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.3_level_array.std140.mat3x2
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.basic_unsized_array.scalar.mat3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.scalar.column_major_mat2x3_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.scalar.mat4x3
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.instance_array_basic_type.std140.mat4x2
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_basic_types.per_block_buffer.scalar
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_nested_struct.per_block_buffer.scalar_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.multi_nested_struct.single_buffer.scalar_comp_access
Stack trace: Fail

FAILURE: dEQP-VK.ssbo.layout.random.16bit.all_shared_buffer.40
Stack trace: Fail
...
```
</p></details></li>

<li>Skipped: 24730/59723 (41.4%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit ce779e1f2ab6b2e81ceba194e5b41d6409f34e67
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev a8cf07c212059bae9d018c05fc896e1115a2f077</li>
<li>SPVGEN: dev 4846370016de9775b7518b51c08260387a7367c4</li>
<li>XGL: dev d3cc9bf1baae76359c2ebab6e7122e6b9c428455</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 20799acf951a8e0b79d95154ecfca851a2f63b98</li>
</ul></details>

<details><summary>CTS tests (Failed: 3/238988)</summary><ul><li>Built with version 1.2.5.1</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 31351/59723 (52.5%)</li>
<li><details><summary>Failed: 1/59723 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.creation_feedback.graphics_tests.vertex_stage_fragment_stage_delayed_destroy
Stack trace: Flake
```
</p></details></li>

<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 31352/59723 (52.5%)</li>
<li>Failed: 0/59723 (0.0%)</li>
<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 31516/59819 (52.7%)</li>
<li>Failed: 0/59819 (0.0%)</li>
<li>Skipped: 28303/59819 (47.3%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 34992/59723 (58.6%)</li>
<li><details><summary>Failed: 2/59723 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.fragment_shading_rate.conservativeunder.dynamic.noattachment.shaderrate.keep.keep.33x35.samples1.vs
Stack trace: Flake

FAILURE: dEQP-VK.pipeline.creation_feedback.graphics_tests.vertex_stage_fragment_stage_no_cache
Stack trace: Flake
```
</p></details></li>

<li>Skipped: 24729/59723 (41.4%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit f9d14ad664625b0ed1e8a650c85b9df3f5d83875
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev a8cf07c212059bae9d018c05fc896e1115a2f077</li>
<li>SPVGEN: dev 4846370016de9775b7518b51c08260387a7367c4</li>
<li>XGL: dev d3cc9bf1baae76359c2ebab6e7122e6b9c428455</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 20799acf951a8e0b79d95154ecfca851a2f63b98</li>
</ul></details>

<details><summary>CTS tests (Failed: 1/238988)</summary><ul><li>Built with version 1.2.5.1</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 31352/59723 (52.5%)</li>
<li>Failed: 0/59723 (0.0%)</li>
<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 31352/59723 (52.5%)</li>
<li>Failed: 0/59723 (0.0%)</li>
<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 31516/59819 (52.7%)</li>
<li>Failed: 0/59819 (0.0%)</li>
<li>Skipped: 28303/59819 (47.3%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 34993/59723 (58.6%)</li>
<li><details><summary>Failed: 1/59723 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.creation_feedback.graphics_tests.vertex_stage_fragment_stage_delayed_destroy
Stack trace: Flake
```
</p></details></li>

<li>Skipped: 24729/59723 (41.4%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 335d7a1d9ea279a426d02cda4cc42fa74d302f70
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev a8cf07c212059bae9d018c05fc896e1115a2f077</li>
<li>SPVGEN: dev 4846370016de9775b7518b51c08260387a7367c4</li>
<li>XGL: dev 3fab12060336620df61ef0f78064c4a35a7cee9b</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 20799acf951a8e0b79d95154ecfca851a2f63b98</li>
</ul></details>

<details><summary>CTS tests (Failed: 1/238988)</summary><ul><li>Built with version 1.2.5.1</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 31352/59723 (52.5%)</li>
<li>Failed: 0/59723 (0.0%)</li>
<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 31352/59723 (52.5%)</li>
<li>Failed: 0/59723 (0.0%)</li>
<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 31515/59819 (52.7%)</li>
<li><details><summary>Failed: 1/59819 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.creation_feedback.graphics_tests.vertex_stage_fragment_stage_delayed_destroy
Stack trace: Flake
```
</p></details></li>

<li>Skipped: 28303/59819 (47.3%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 34994/59723 (58.6%)</li>
<li>Failed: 0/59723 (0.0%)</li>
<li>Skipped: 24729/59723 (41.4%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 83cdd91fe7f285392b756e4db439817408c7714b
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev a8cf07c212059bae9d018c05fc896e1115a2f077</li>
<li>SPVGEN: dev 4846370016de9775b7518b51c08260387a7367c4</li>
<li>XGL: dev 3fab12060336620df61ef0f78064c4a35a7cee9b</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 20799acf951a8e0b79d95154ecfca851a2f63b98</li>
</ul></details>

<details><summary>CTS tests (Failed: 1/238988)</summary><ul><li>Built with version 1.2.5.1</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 31352/59723 (52.5%)</li>
<li>Failed: 0/59723 (0.0%)</li>
<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 31351/59723 (52.5%)</li>
<li><details><summary>Failed: 1/59723 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.creation_feedback.graphics_tests.vertex_stage_fragment_stage_no_cache
Stack trace: Flake
```
</p></details></li>

<li>Skipped: 28371/59723 (47.5%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 31457/59819 (52.6%)</li>
<li>Failed: 0/59819 (0.0%)</li>
<li>Skipped: 28362/59819 (47.4%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 34994/59723 (58.6%)</li>
<li>Failed: 0/59723 (0.0%)</li>
<li>Skipped: 24729/59723 (41.4%)</li>
</ul></details>


</ul></details>

File: llpc/test/shaderdb/ObjStorageBlock_TestMemoryQualifier_lit.frag [hunk 1]
```diff
@@ -22,7 +22,7 @@ void main()
 
 ; SHADERTEST-LABEL: {{^// LLPC}} SPIR-V lowering results
 ; SHADERTEST: %{{[0-9]*}} = call i8 addrspace(7)* (...) @lgc.create.load.buffer.desc.{{[0-9a-z]*}}(i32 1, i32 0, i32 0,
-; SHADERTEST: %{{[0-9]*}} = load atomic <4 x float>, <4 x float> addrspace(7)* %{{[0-9]*}} unordered, align 16
+; SHADERTEST: %{{[0-9]*}} = load atomic float, float addrspace(7)* %{{[0-9a-z.]*}} unordered, align 4
```
Comment (kuhar) [inline llpc/test/shaderdb/ObjStorageBlock_TestMemoryQualifier_lit.frag:25]: `.` matches every character. To allow for names with `.` in them, I think you'd want to use `\.` instead.

Comment (AMD-dwang) [reply llpc/test/shaderdb/ObjStorageBlock_TestMemoryQualifier_lit.frag:25]: Only digits will be matched here. I change it to only match digits. Thanks.

**Comment by amdvlk-admin** [comment]
### Test summary for commit a124cbd0a4969038bd96c9ee0692cdab2b1a4e18
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev a8cf07c212059bae9d018c05fc896e1115a2f077</li>
<li>SPVGEN: dev 4846370016de9775b7518b51c08260387a7367c4</li>
<li>XGL: dev 3fab12060336620df61ef0f78064c4a35a7cee9b</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 20799acf951a8e0b79d95154ecfca851a2f63b98</li>
</ul></details>

<details><summary>CTS tests (Failed: 1/238295)</summary><ul><li>Built with version 1.2.5.1</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 30790/59551 (51.7%)</li>
<li>Failed: 0/59551 (0.0%)</li>
<li>Skipped: 28761/59551 (48.3%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 30790/59551 (51.7%)</li>
<li>Failed: 0/59551 (0.0%)</li>
<li>Skipped: 28761/59551 (48.3%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 30904/59642 (51.8%)</li>
<li>Failed: 0/59642 (0.0%)</li>
<li>Skipped: 28738/59642 (48.2%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 34332/59551 (57.7%)</li>
<li><details><summary>Failed: 1/59551 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.draw.shader_viewport_index.fragment_shader_implicit
Stack trace: Fail
```
</p></details></li>

<li>Skipped: 25218/59551 (42.3%)</li>
</ul></details>


</ul></details>

**Comment by JaxLinAMD** [comment]
retest this please ( <-- token to trigger jenkins test)

**Comment by amdvlk-admin** [comment]
### Test summary for commit a124cbd0a4969038bd96c9ee0692cdab2b1a4e18
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev a8cf07c212059bae9d018c05fc896e1115a2f077</li>
<li>SPVGEN: dev 4846370016de9775b7518b51c08260387a7367c4</li>
<li>XGL: dev 3fab12060336620df61ef0f78064c4a35a7cee9b</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 20799acf951a8e0b79d95154ecfca851a2f63b98</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/238295)</summary><ul><li>Built with version 1.2.5.1</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 30790/59551 (51.7%)</li>
<li>Failed: 0/59551 (0.0%)</li>
<li>Skipped: 28761/59551 (48.3%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 30790/59551 (51.7%)</li>
<li>Failed: 0/59551 (0.0%)</li>
<li>Skipped: 28761/59551 (48.3%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 30904/59642 (51.8%)</li>
<li>Failed: 0/59642 (0.0%)</li>
<li>Skipped: 28738/59642 (48.2%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 34333/59551 (57.7%)</li>
<li>Failed: 0/59551 (0.0%)</li>
<li>Skipped: 25218/59551 (42.3%)</li>
</ul></details>


</ul></details>

**Comment by JaxLinAMD** [comment]
@kuhar @amdrexu  ping for code review.

**Review by amdrexu — APPROVED** [review]

**Pull request merged**